diff --git a/scratch/CMakeLists.txt b/scratch/CMakeLists.txt
index 7d05f6022..218764000 100644
--- a/scratch/CMakeLists.txt
+++ b/scratch/CMakeLists.txt
@@ -85,7 +85,7 @@ foreach(subdir ${scratch_subdirectories})
   else()
     # Otherwise we pick all the files in the subdirectory
     # and create a scratch for them automatically
-    file(GLOB scratch_sources CONFIGURE_DEPENDS ${subdir}/[^.]*.cc)
+    file(GLOB scratch_sources CONFIGURE_DEPENDS ${subdir}/[^.]*.cc "./helper/*.cc")
     create_scratch("${scratch_sources}")
   endif()
 endforeach()
